### ISSUE TITLE
[Discussion] 

### ISSUE THREAD

**Issue by gcruchon** [AxaGuilDEv/react-toolkit#768]
# Communication improvement 

Hello folks,

First, I think it would be a good idea to have a "discussion" label for issues which deals with chats about roadmaps, some new ideas, etc...

Then, let's face it, many contribution are coded outside working schedules. Is there a way to reach asynchronously the cummunity (repo maintainers, but also all contributors?). AXA's MS Teams does not seem appropriate here (IMHO).

**Comment by xballoy** [comment]
Good idea, I added a `Type: Discussion` label.

To communicate asynchronously the best way is to use the Github issues as many open source project.

**Comment by gcruchon** [comment]
I agree, but sometimes you need a bit more (something closer to "instant messenging" - slack, teams, ...)

**Comment by arnaudforaison** [comment]
For the moment, we are only able to discuss with issues. If team grow & find more time, maybe we can add an instant messenger.

**Comment by xballoy** [comment]
I'm not sure that adding a new way to communicate will improve it, on the contrary some channel will probably never receive any answer.
Furthermore, if everything is here we can have an history of the discussion and it's public.

**Comment by arnaudforaison** [comment]
@youf-olivier & @samuel-gomez-axa , your opinion ?

**Comment by gcruchon** [comment]
The instant messenging would also be public and archived, @xballoy :)
